hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/spi/state/impl/StatesImpl.java

```diff
@@ -39,13 +44,17 @@ public void updateChildren(final ServicesState sourceState) {
     }
 
     @Override
-    public @Nonnull <K, V> State<K, V> get(@Nonnull final String stateKey) {
+    public @Nonnull <K, V> State<K, V> get(@Nonnull final StateKey stateKey) {
         Objects.requireNonNull(stateKey);
 
-        if (stateKey.equals(ACCOUNTS_KEY)) {
+        if (stateKey.equals(ACCOUNT_STORE)) {
             final var state =
                     new InMemoryStateImpl<>(stateKey, children.accounts(), children.signedAt());
             return (StateBase) state;
+        } else if (stateKey.equals(ALIASES_STORE)) {
+            final var state =
+                    new RebuiltStateImpl<>(stateKey, children.aliases(), children.signedAt());
+            return (StateBase) state;
         }
         // Will be adding other keys as needed. This is the only key needed for signature
         // verification.
```

Comment:
It'd probably be easier to figure out what happened if we threw an exception for the other types of state keys:
```
} else {
    throw new NotImplementedException(String.format("State key %s not supported", stateKey));
}
```

Reply:
Fixed. Thanks !